gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -2035,3 +2010,47 @@ def test_initialize_client_w_{{transport_name}}():
     assert client is not None
 
 {% endmacro %}
+
+{# rest_method_bad_request_test generates tests for rest methods
+ # which raise a google.api.core.exceptions.BadRequest error.
+ # TODO(https://github.com/googleapis/gapic-generator-python/issues/2157): Refactor this macro to include `gRPC` coverage.
+#}
+{% macro rest_method_bad_request_test(service, method, transport, is_async) %}
```

Comment:
non-blocking, but desirable: Same comment as in the other PR: I'd have  a macro-level fail-safe check that `"rest" in transport` that will cause a generation-time exception if we call the macro for non-REST. We can adapt/remove that as we factor gRPC functionality in here.